### PR TITLE
Release 0.3.0

### PR DESCRIPTION
**Pull request by mocsharp**
### Description

Release 0.3.0

### Status
**Ready**

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] All tests passed locally.
- [ ] [Documentation comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments) included/updated.
- [ ] User guide updated.
- [ ] I have updated the changelog

### CODE REVIEW

File: docs/api/rest/fhir.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  ~ limitations under the License.
+-->
+
+# DICOMWeb STOW-RS APIs
+
+The `fhir/` endpoint implements the specifications defined in [section 3.1.0 RESTful API](http://hl7.org/implement/standards/fhir/http.html)
+defined by HL7 (Health Level 7 International) to enable triggering new workflows. The FHIR service supports multiple versions of the Fast Healthcare Interoperability Resources (FHIR) specifications published by Health Level 7 International (HL7).
+
+[!Note]
+The service does not support `CapabilityStatement` at this moment and does not group the incoming FHIR resources. Therefore, each incoming FHIR resource will trigger a new workflow request.
```
Comment (Coco-Ben) [inline docs/api/rest/fhir.md:23]: Suggest linking to documentation for CapabilityStatement (either in this documentation or in the h17 docs).

File: docs/api/rest/fhir.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+| Code | Data Type                                                     | Description                                                           |
+| ---- | ------------------------------------------------------------- | --------------------------------------------------------------------- |
+| 201  | Original JSON or XML document.                                | Resource created & stored successfully.                               |
+| 400  | [OperationOutcome](http://hl7.org/fhir/operationoutcome.html) | Unable to parse the resource or mismatching resource type specified.. |
+| 415  | `none`                                                        | Unsupported media type                                                |
+| 500  | [OperationOutcome](http://hl7.org/fhir/operationoutcome.html) | Server error.                                                         |
+
+[!Note]
+The `Location` header in the response given that the resources created are for inference only.
```
Comment (Coco-Ben) [inline docs/api/rest/fhir.md:66]: This is a bit confusing. Is the header given *when* the resources are created?

File: docs/api/rest/fhir.md (hunk 1 above)
Comment (mocsharp) [reply docs/api/rest/fhir.md]: Linked to HL7.org

File: docs/api/rest/fhir.md (hunk 2 above)
Comment (mocsharp) [reply docs/api/rest/fhir.md]: Removed the note as a copy&paste error

**Review by dbericat — APPROVED** [review]
LGTM

**Pull request merged**